packages/styles/scss/components/content-switcher/_content-switcher.scss

```diff
@@ -52,15 +52,33 @@
     color: $text-secondary;
     text-align: left;
     text-decoration: none;
-    transition: all $duration-fast-01 motion(standard, productive);
+    transition: all $duration-moderate-01 motion(standard, productive);
     white-space: nowrap;
 
+    &::after:not(&:disabled) {
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      height: 100%;
+      background-color: transparent;
+      content: '';
+      transform: scaleY(0);
+      transform-origin: bottom;
+      transition: all $duration-moderate-01 carbon--motion(standard, productive);
```

Comment:
should be just `motion(standard, productive)` not `carbon--motion(standard, productive)`

Reply:
```suggestion
      transition: all $duration-moderate-01 motion(standard, productive);
```